code-challenges.js

```diff
@@ -8,54 +8,185 @@
+
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
 // Pseudo code:
-
-const myArray = [13, 34, 5, 10, 27, 42]
+//problem: the numbers in the array need to be identified as odd or even and returned as such.
+//function expression: using a for loop, index the list of numbers and then have the new indexed array % 2 === 0 for the even numbers. For the odd numbers, the indexed array $ 2 == 1. 
+//set parameters: myArray[i] % 2 === 0; myArray[i] % 2 == 1. 
+//make decision: write a for loop indexing starting with i=0, ending at i<=42 (largest number in the array). Then write an if for the odd numbers and another if function for the even numbers. If the number is odd or even return the appropriate word ("even" or "odd").
+//return: "even" "odd"
+
+const myArray = [13, 34, 5, 10, 27, 42] 
+for(let i=0; i<=42; i++){
+    if(myArray[i] % 2 === 0){
+        console.log('"even"')
+    }
+    if(myArray[i] % 2 == 1){
+        console.log('"odd"')
+    }
+}
+//Output: "odd" "even" "odd" "even" "odd" "even"
+//I wasn't sure if the quotations needed to be included in the final output so I added them in to match the provided output.
 
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
 // Pseudo code:
+// problem: Given two numbers, the smaller number needs to be subtracted from the larger number and the output is the difference.
+// function expression: if statement for the first possible subtraction combination number1 > number2; if else for the second subtraction combination number2 > number1; else for if the numbers are equal.
+// set parameters: if (number1 is > than number2){console.log (number1 - number2)} if else (number2 > number1){console.log(number2 - number1)}
+// make decision: write if statements for both sets of numbers. write an else statement if the variables are equal to one another.
+// return: 42 (set one), 3 (set two)
+
 
 // Set one:
 const number1 = 58
 const number2 = 100
 
+if(number1 > number2){
+    console.log(number1 - number2)
+} else if(number2 > number1){
+    console.log(number2 - number1)
+} else {
+    console.log("Try Again")
+}
+
 // Set two:
 const number3 = 27
 const number4 = 24
+
+if(number3 > number4){
+    console.log(number3 - number4)
+} else if(number4 > number3){
+    console.log(number4 - number3)
+} else {
+    console.log("Try Again")
+}
+
+//Output: 42,3
+//I was able to get the logic on this problem and figure out the equation to use. I started with plugging in a simple equation (did not think it would work) before trying other processes.
```

Comment:
Logic looks right on this problem.  Next time, try to place this logic into a function to make it more dynamic.